opennebula/resource_opennebula_template_test.go

```diff
@@ -164,6 +167,8 @@ resource "opennebula_template" "template" {
 	boot = ""
   }
 
+  labels = "test1,test2,test3"
```

Comment:
An interesting test to check is the label "hierarchy" thanks to `/` separator.
Can you add this in thest to be sure hierarchy works